appcelerator/titanium_mobile
# [TIMOB-15061] (3_1_X) iOS: Adding support for the new naming convention in Xcode 5's retina sim

Backport of #4655
**Testing instruction** 

Should be able to properly switch between 4inch and 3.5 inch iphone sim and ipad retina properly using studio. 
follow additional instructions in [JIRA](https://jira.appcelerator.org/browse/TIMOB-15061)
